core/templates/components/summary-tile/topic-summary-tile.component.html

```diff
@@ -11,7 +11,12 @@
       </picture>
     </div>
     <div class="topic-details" [ngStyle]="{'background-color': getDarkerThumbnailBgColor()}" ng-class="{'mask-effect': !isPublished}">
-      <span class="topic-name">{{ topicSummary.getName() }}</span>
+      <span class="topic-name" *ngIf="!isTopicTranslationKeyToBeDisplayed()">
+        {{ topicSummary.getName() }}
+      </span>
+      <span class="topic-name" *ngIf="!isTopicTranslationKeyToBeDisplayed()">
```

Comment:
```suggestion
      <span class="topic-name" *ngIf="isTopicTranslationKeyToBeDisplayed()">
```

Looks like a mistake. Have you check that these functionality is working as expected after making these changes?

Reply:
Yes, it was a mistake, was about to push this fix. Thanks for pointing it out!

Reply:
Done!